doc/continuous-integration.md

```diff
@@ -118,6 +118,21 @@ echo -n "super-secret" | \
   --project github.com/SovereignCloudStack/zuul-config
 ```
 
+### Job customization
+
+In some cases you may want to change your `environment.tfvars` file before running the e2e test without changing
+the `environment.tfvars` file in the repository. For example, you may want to change the `image` variable to test a different
+system image without altering the default image used by the project.
+
+To do so, you can in the body of the PR add the following text:
+```text
+
+```ZUUL_CONFIG
+image = "Ubuntu 20.04"
+```.
+
+```
+The dot at the end of the code block is just for formatting purposes and should not be included in the PR.
```

Comment:
Instead of using a period (dot) at the end of the code block, you can opt for a 4-space indentation for the inner block. Take a look at the raw version of the following:

```text
    ```ZUUL_CONFIG
    image = "Ubuntu 20.04"
    ```
```